cli.js

```diff
@@ -1,12 +1,40 @@
 #!/usr/bin/env node
-import fs from 'node:fs'
 import { globbySync } from 'globby'
+import fs from 'node:fs'
 import sortPackageJson from './index.js'
 
 const isCheckFlag = (argument) => argument === '--check' || argument === '-c'
+const isHelpFlag = (argument) => argument === '--help' || argument === '-h'
+const isVersionFlag = (argument) =>
+  argument === '--version' || argument === '-V'
 
 const cliArguments = process.argv.slice(2)
 const isCheck = cliArguments.some(isCheckFlag)
+const isHelp = cliArguments.some(isHelpFlag)
+const isVersion = cliArguments.some(isVersionFlag)
+
+if (isHelp) {
+  console.log(
+    `Usage: sort-package-json [OPTION...] [FILE...]
+Sort npm package.json files. Default: ./package.json
+Strings passed as files are parsed as globs.
+
+  -c, --check                check if FILES are sorted
+  -q, --quiet                don't output success messages
+  -h, --help                 display this help and exit
+  -V, --version              display the version and exit
```

Comment:
Why don't we use lowercased `-v`?

Reply:
The standard for cli tools and commands is that lowercase -v means verbose and uppercase -V means version

Reply:
I don't think that's true, https://nodejs.org/api/cli.html#-v---version

Reply:
it seems there is not necessarily a standard on paper from my research, but in my usage of linux, ive generally noted -V to be version and -v to be verbose. if we ever plan to add a verbosity switch, we should leave version as -V, but if you still think it should be lowercase, i'll change it

Reply:
From my experience, many tools uses `-v` for version. Let's take it, and not worry about `--verbose` for now.